test/unit/Markdown.test.js

```diff
@@ -0,0 +1,77 @@
+import Markdown from '../../src/components/Markdown.react.js';
+import React from 'react';
+import {shallow, render} from 'enzyme';
+
+test('Input renders', () => {
+    const md = render(<Markdown />);
+
+    expect(md.html()).toBeDefined();
+});
+
+describe('dedent', () => {
+    const md = shallow(<Markdown />).instance();
+
+    test('leading spaces and tabs are removed from a single line', () => {
+        [
+            'test',
+            ' test',
+            '        test',
+            '\t\t\ttest',
+            '    \t    test',
+            '\t    \ttest',
+        ].forEach(s => {
+            expect(md.dedent(s)).toEqual('test');
+        });
+
+        expect(md.dedent('    test    ')).toEqual('test    ');
+    });
+
+    test('same chars are removed from multiple lines, ignoring blanks', () => {
+        ['', '    ', '\t', '\t\t', '          ', '\t    \t'].forEach(pre => {
+            expect(
+                md.dedent(
+                    pre +
+                        'a\n' +
+                        pre +
+                        '  b\r' +
+                        pre +
+                        'c\r\n' +
```

Comment:
Prettier certainly did not improve clarity here... and its indentation choice is peculiar for this case. Oh well...

Reply:
cannot partially disable just for this chunk?